meta-resin-jethro/recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -1,5 +1,7 @@
 PACKAGES =+ "${PN}-iwlwifi-7265D-10"
-
 RDEPENDS_${PN}-iwlwifi-7265D-10 = "${PN}-iwlwifi-license"
-
 FILES_${PN}-iwlwifi-7265D-10 = "/lib/firmware/iwlwifi-7265D-10.ucode"
+
+PACKAGES =+ "${PN}-iwlwifi-8000C-13"
+RDEPENDS_${PN}-iwlwifi-8000C-13 = "${PN}-iwlwifi-license"
+FILES_${PN}-iwlwifi-8000C-13 = "/lib/firmware/iwlwifi-8000C-13.ucode"
```

Comment:
https://github.com/resin-os/meta-resin/pull/230/commits/4c78c65fabf5414515e83d2fcfb01af46403e3ed

The CONNECTIVITY_FIRMWARES_append is used for this.